src/mapml/layers/TemplatedTileLayer.js

```diff
@@ -33,6 +33,9 @@ export var TemplatedTileLayer = L.TileLayer.extend({
       L.TileLayer.prototype.onAdd.call(this,this._map);
       this._handleMoveEnd();
     },
+    addTo: function(map){
```

Comment:
This function is unnecessary, because it overrides an ancestor function of the same name, and is probably not doing what it's supposed to: it is creating a whole new Leaflet TileLayer in place of the current TemplatedTileLayer, using the factory L.tileLayer method.  Although it's unnecessary, if we wanted to call the ancestor addTo function ourselves, as we do in the initialize constructor, we could use the L.TileLayer.prototype.addTo.call(this, map) technique described [here](https://leafletjs.com/examples/extending/extending-1-classes.html#methods-of-the-parent-class).